config/prometheus/monitor.yaml

```diff
@@ -5,16 +5,16 @@ kind: ServiceMonitor
 metadata:
   labels:
     control-plane: controller-manager
-  name: controller-manager-metrics-monitor
-  namespace: system
+  name: lvm-operator-controller-manager-metrics-monitor
+  # namespace: system
```

Comment:
Initially I thought, since our lvm-operator namespace is different (not system it is called lvm-operator-system), removed the namespace part. But now I understand that all the resources which has `system` namespace is deployed correctly into the right operator namespace.
Added it back